tests/phpunit/ExceptionConverterTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor\Tests;
+
+use Keboola\FtpExtractor\Exception\ApplicationException;
+use PHPUnit\Framework\TestCase;
+use Keboola\FtpExtractor\Exception\ExceptionConverter;
+use League\Flysystem\FileNotFoundException;
+use Keboola\Component\UserException;
+
+class ExceptionConverterTest extends TestCase
+{
+    /**
+     * @dataProvider exceptionMessageProvider
+     */
+    public function testHandleCopyFilesExpectedUserException(string $message): void
```

Comment:
Imho bys mohl mít všechny vstupy v tom dataprovideru - `$expectedExceptionClass, $expectedExceptionMessage, $thrownException`

A pak bys měl jen jeden testcase na každou tu handle metodu... 
```
$this->expectException($expectedExceptionClass);
        $this->expectExceptionMessage($expectedExceptionMessage);
        try {
            throw $thrownException;
        } catch (\Throwable $e) {
            ExceptionConverter::handleCopyFilesException($e);
        }
```

Reply:
Ano tím se to dost zjednodušilo, mrkni na to :)